recipes/cuda-histogram/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "cuda_histogram" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3672c474dd094e2fefd364757d8fb6aa016af6e2ffee356e2a9a78c88f58690
+  patches:
+    - cupy.patch
```

Would suggest adding a comment here as a reminder to future readers what this patch is used for

```suggestion
    # Patch `pyproject.toml`'s requirement on `cupy-cuda12x` to `cupy` to fix `pip check`.
    - cupy.patch
```

---

Thanks!